### PR TITLE
Checkout Pixel Validation Options page

### PR DESCRIPTION
**Pull request by fabrice-olapic**
Version 1 of the Checkout Pixel Validation Page.

### CODE REVIEW

File: checkout-pixel-validation-guide.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+6. **Right click the request, and click Save as HAR with Content**
+
+	Please send the .HAR file to your Solution Architect so that we can
+	further inspect.
+
+	![Image 5](http://olapic-data.s3.amazonaws.com/fabrice/cpvalidation/image5.png)
+
+7. **Analyze the checkout.gif request**
+
+	Search for data.photorank.me in the Search field in the Network tab of
```
Comment (jhankim) [inline checkout-pixel-validation-guide.md]: For URLs, code snippets inside documents, it's super helpful to use Markdown's inline code syntax (surround the word in backticks)

File: checkout-pixel-validation-guide.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+	the Dev Tools. Take a close look at the Query String Parameters section
+	of the request. This is where you will validate the data being passed to
+	Olapic's Analytics AP:
```
Comment (jhankim) [inline checkout-pixel-validation-guide.md]: Misspelling: `AP` => `API`

File: checkout-pixel-validation-guide.md [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+
+-   Check the *`auth_token`*. This value should match your Olapic API key
+    (Settings &gt; Account Basics)
+
+-   Check the *`analytics_id`*. This value should match the analytics_id
+    from Step 1.
+
+-   Check the *`product[]`* parameters.
```
Comment (jhankim) [inline checkout-pixel-validation-guide.md]: Change
> Check the *`product[]`* parameters.

to:
> Check the *`product[]`* parameters for correct product ID and total value.

File: checkout-pixel-validation-guide.md [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+
+        -   `Product_ID123:50`
+
+    -   Or this can be itemized:
+
+        -   `Product_ID123:25`
+
+        -   `Product_ID123:25`
+
+-   Check the *currency_code *parameter. You should see the ISO 4217
```
Comment (jhankim) [inline checkout-pixel-validation-guide.md]: Change
> Check the *currency_code *parameter.

to:
> Check the *currency_code* parameter.

File: checkout-pixel-validation-guide.md [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+Thereafter please provide the following information to your Solutions
+Architect so we can verify this data on our end:
+-   Sample URL where an Olapic widget is implemented.
+
+-   The products that were added to the cart.
+
+-   URL where the Olapic checkout code is implemented.
+
+-   .HAR data from confirmation page.
+
```
Comment (jhankim) [inline checkout-pixel-validation-guide.md]: Remove extra line after every bullet item.

File: checkout-pixel-validation-guide.md [hunk 6]
```diff
@@ -0,0 +1,150 @@
...
+        -   `Product_ID123:25`
+
+-   Check the *currency_code *parameter. You should see the ISO 4217
+    Alphabetic code of the currency. (e.g. Use 'EUR' if the Amount value
+    is in Euro, ‘USD’ for US Dollars).
+
+![Image 6](http://olapic-data.s3.amazonaws.com/fabrice/cpvalidation/image6.png)
+
+Thereafter please provide the following information to your Solutions
```
Comment (jhankim) [inline checkout-pixel-validation-guide.md]: I feel like the word usage of "thereafter" is sort of weird here. I think taking it out would make more sense. If we're going to keep it, add a comma.

File: checkout-pixel-validation-guide.md [hunk 7]
```diff
@@ -0,0 +1,150 @@
...
+==========
+
+**Table of Contents**
+
+- [Overview](#overview)
+- [Validation Options](#validation-options)
+
+## Overview
+
+The Olapic Checkout Pixel allows you to send important data from your conversion funnel to Olapic. The purpose of this code is to capture the data from the orders made by customers, and tie the data back to Olapic components or widgets installed on your site.
```
Comment (jhankim) [inline checkout-pixel-validation-guide.md]: Remove comma after `... made by customers`

**Comment by fabrice-olapic** [comment]
hey @jhankim, thanks for reviewing, I addressed all the issues you brought up, please take a look. Thanks.

**Comment by jhankim** [comment]
🚢

**Pull request merged**